src/utils/getCadOutput.m

```diff
@@ -54,7 +54,8 @@
         return
         
     else
-        warning('Cache file exists but might be outdated. Ignoring it.')
+        warning(strcat('Cache file exists but might be outdated. ',...
+            ' Ignoring it and generating new one.'));
```

Comment:
check spaces

Reply:
which spaces. I did ctr+I, let me know if you are talking about something else

Reply:
warning(['Cache file exists but might be outdated. ',...
            'Ignoring it and generating new one.']);

Reply:
I have changed it as above